download/nightly/index.md

```diff
@@ -62,12 +87,24 @@ a certain build is not available for your operating system, please check the pre
     }
 
     function showError() {
-      document.getElementById("nightly-loading").textContent = "Unable to retrieve Nightly Builds. Please try again later.";
+      const nightlyLoading = document.getElementById("nightly-loading");
+      nightlyLoading.textContent = "Unable to retrieve Nightly Builds. Please try again later.";
+      const buildDirs = document.createElement("div");
+      buildDirs.innerHTML = document.getElementById("build-dirs").innerHTML;
+      nightlyLoading.parentNode.insertBefore(buildDirs, nightlyLoading);
```

Comment:
Hmm the error message is a bit easy to miss when you’re expecting the list view… I was actually kinda confused for a moment when I briefly lost my wifi connection and got the table instead.

Reply:
Error messages have been adjusted to be more prominent and informative.